flocker/node/agents/cinder.py

```diff
@@ -291,8 +326,8 @@ def attach_volume(self, blockdevice_id, attach_to):
     def detach_volume(self, blockdevice_id):
         our_id = self.compute_instance_id()
         try:
-            nova_volume = self.nova_volume_manager.get(blockdevice_id)
-        except NovaNotFound:
+            cinder_volume = self.cinder_volume_manager.get(blockdevice_id)
+        except CinderNotFound:
```

Comment:
Hm.  Does this make any real difference?

If we stick with this, we no longer use `get` from the nova volume manager so maybe we should drop that from the interface.